$MirOS: ports/www/links+/patches/patch-config_h_in,v 1.1.7.1 2005/03/18 15:59:38 tg Exp $
$OpenBSD: patch-config_h_in,v 1.1 2002/12/17 08:09:23 fgsch Exp $
--- config.h.in.orig	Thu Feb 24 16:18:48 2005
+++ config.h.in	Mon May 23 22:12:23 2005
@@ -378,6 +378,12 @@
 /* */
 #undef HAVE_CHMOD
 
+/* Define if you have the MD5Data function.  */
+#undef HAVE_MD5DATA
+
+/* Define if you have the MD5Init function.  */
+#undef HAVE_MD5INIT
+
 /* */
 #undef HAVE_ALARM
 
